packages/graphiql/cypress/e2e/docs.cy.ts

```diff
@@ -1,7 +1,7 @@
 import { version } from 'graphql/version';
 
 beforeEach(() => {
-  cy.visit('/');
+  cy.visit('');
```

Comment:
why this change? cypress doesn't recommend this, an recommend use `/`, is much more understandable and previously worked fine

Reply:
has to do with using the dev build, pushing a fix that satisfies this reccomendation now